submissions/hannasyn/js-dom/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>DOM</title>
+  </head>
+  
+  <body>
+    <header class="header">
+      <h1 class="title">FIVE SHORT FACTS ABOUT <span>FEMINISM</span> YOU SHOULD DEFINITELY KNOW!</h1>
+    </header>
+    <div class="wrapper">
+      <aside class="aside">
+        <a class="aside-btn" href="#"></a>
```

Comment:
Why you use HTML tag - link for button functionality?

Reply:
Fixed